File: src/main/java/no/nav/veilarboppfolging/controller/response/OppfolgingStatus.java
package no.nav.veilarboppfolging.controller.response;




import java.time.LocalDate;
import java.time.ZonedDateTime;
import java.util.List;


public class OppfolgingStatus {
    public String fnr;
    public String aktorId;
    public String veilederId;
    public boolean reservasjonKRR;
    public boolean registrertKRR;
    public boolean kanVarsles;
    public boolean manuell;
    public boolean underOppfolging;
    public boolean underKvp;

    public ZonedDateTime oppfolgingUtgang;
    private boolean kanStarteOppfolging;
    @Deprecated
    private AvslutningsStatusDto avslutningStatus;
    private List<OppfolgingPeriodeDTO> oppfolgingsPerioder;
    private boolean harSkriveTilgang;
    private Boolean inaktivIArena;
    private Boolean kanReaktiveres;
    public LocalDate inaktiveringsdato;
    /*
    * Får treff i "mulighetsrommet", "arbopp", "arbopp-new"
    * */
    private Boolean erSykmeldtMedArbeidsgiver;
    private String servicegruppe;
    private String formidlingsgruppe;
    private String rettighetsgruppe;
}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/response/OppfolgingStatus.kt
package no.nav.veilarboppfolging.controller.response

import java.time.LocalDate
import java.time.ZonedDateTime


class OppfolgingStatus(
    val fnr: String,
    val aktorId: String,
    val veilederId: String?,
    val reservasjonKRR: Boolean,
    val registrertKRR: Boolean,
    val kanVarsles: Boolean,
    val manuell: Boolean,
    val underOppfolging: Boolean,
    val underKvp: Boolean,
    val oppfolgingUtgang: ZonedDateTime?,
    val kanStarteOppfolging: Boolean?,
    @Deprecated("")
    val avslutningStatus: AvslutningsStatusDto? = null,
    val oppfolgingsPerioder: List<OppfolgingPeriodeDTO>,
    val harSkriveTilgang: Boolean?,
    val inaktivIArena: Boolean?,
    val kanReaktiveres: Boolean,
    var inaktiveringsdato: LocalDate? = null,
    /*
    * Får treff i "mulighetsrommet", "arbopp", "arbopp-new"
    * */
    val erSykmeldtMedArbeidsgiver: Boolean,
    val servicegruppe: String,
    val formidlingsgruppe: String,
    val rettighetsgruppe: String,
)